src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -603,7 +607,7 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
             switch (cfg[0]) {
                 case "config":
                     if (cfg.length < 3) {
-                        logger.warn("NODE {}: Configuration invalid {}", nodeId, configurationParameter.getKey());
+                        logger.debug("NODE {}: Configuration invalid {}", nodeId, configurationParameter.getKey());
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn.